tests/test_lochness.py

```diff
@@ -115,6 +115,15 @@ def update_for_redcap(self, study):
 
         self.write_keyring_and_encrypt()
 
+    def update_for_ampsz_redcap(self, study):
+        token = Tokens()
+        api_token, url = token.read_token_or_get_input('redcap_ampsz')
```

Comment:
Is it okay that you missed `c` between `sz` in `redcap_ampsz`?

Reply:
This is calling a test method which I've named to have "SZ", so it works.